src/js/controller/controller.js

```diff
@@ -248,6 +248,16 @@ Object.assign(Controller.prototype, {
                         index: _model.get('item'),
                         item: playlistItem
                     });
+                    const { title, image } = playlistItem;
+                    if ('mediaSession' in navigator && title && image) {
+                        navigator.mediaSession.metadata = new window.MediaMetadata({
+                            title: title,
+                            artist: window.location.hostname,
+                            artwork: [
+                                { src: image, sizes: '384x384' }
+                            ]
+                        });
+                    }
```

Comment:
This code should go before triggering the playlist item event `_this.trigger(PLAYLIST_ITEM`